### DISCUSSION TITLE
mpv不支持8k视频播放了

### DISCUSSION THREAD

**Discussion by zch2136** [hooke007/mpv_PlayKit#436] [mpv 通用]
自本年度1月2日后版开始至今，点击播放就出现闪退，2023年12月31日之前版本都正常，不知何故

**Comment by hooke007** [comment]
既没有日志；也无法复现
另外用纯净模式测了没

**Comment by zch2136** [comment]
hooke007您好！
       用的测试版本基本是每周稳定版那种，未改变任何参数，测试为同一8K视频，用potplayer和mpcbe播放器都能正常播放，MPV截至现今每次测试（自23年12月31日版本后）也是用纯净版，每次下载原版测试后不行，就直接删除了，一直到现在用的MPV是23年11月5日的稳定版。大神你们播放8K都正常么？？？

**Reply by zhongfly** [reply]
你把log贴出来吧
`mpv --log-file=mpv.log`

**Comment by zch2136** [comment]
好的，谢谢！请指点。
[mpv.log](https://github.com/hooke007/MPV_lazy/files/14811522/mpv.log)

**Reply by hooke007** [reply]
```
[  53.184][d][cplayer] Terminating demuxers...
[  53.185][d][cplayer] Done terminating demuxers.
[  53.185][v][cplayer] finished playback, no audio or video data played (reason 4)
```

不觉得是设置问题，建议向上游报告
